io/pci/PowerVMEEH.py

```diff
@@ -66,7 +66,7 @@ def setUp(self):
         if '0x1' not in genio.read_file(eeh_enable_file).strip():
             self.cancel("EEH is not enabled, please enable via FSP")
         self.max_freeze = self.params.get('max_freeze', default=1)
-        self.pci_addr = [self.params.get('pci_device', default='')]
+        self.pci_addr = [self.params.get('pci_device', default=None)]
```

Comment:
Please have this as a separate PR.
We will need to review Readme file to see if space is what it is separated by.

Reply:
I now have 3 separate commits